stdlib/format.mli

```diff
@@ -439,6 +446,9 @@ val set_max_indent : int -> unit
   ["123456789"] and ["123456789A"] .
   Note also that vertical boxes never fit on a line whereas horizontal boxes
   always fully fit on the current line.
+  Opening a box may split a line whereas the contents may have fit.
+  If this behavior is problematic, it can be curtailed by setting the maximum
+  indentation limit to [margin - 1].
```

Comment:
It might also help to note that setting [maxindent] to [margin] is invalid. At least in my experience, trying that is one of the first things to try to disable the maxindent mechanism.

Reply:
This is a very good point, indeed the impossibility of setting maxindent to margin is potentially surprising.